__tests__/app.test.js

```diff
@@ -433,3 +464,98 @@ describe("GET /api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("POST api/articles/:article_id/comments", () => {
+  test("200: returns object of article with updated votes", () => {
```

Comment:
You should change the test description to be more clear that we are expecting the added comment to be returned